mlflow/__init__.py

```diff
@@ -167,6 +167,10 @@
 last_active_run = mlflow.tracking.fluent.last_active_run
 MlflowClient = mlflow.client.MlflowClient
 
+load_text = mlflow.tracking.fluent.load_text
+load_dict = mlflow.tracking.fluent.load_dict
+load_image = mlflow.tracking.fluent.load_image
```

Comment:
Are we sure this is correct? The proposal in https://github.com/mlflow/mlflow/issues/6428 is to add `mlflow.artifacts.load_text()`, `mlflow.artifacts.load_json()`, and `mlflow.artifacts.load_image()`.

Reply:
Sorry, my bad. I started right but ended in different direction. Have made the changes accordingly ! Thanks for correction :)